README.md

```diff
@@ -21,44 +21,59 @@ features like flow control, data structures, and functions, with high-level
 constructs like objects, protocols, closures, and generics. Swift embraces
 modules, eliminating the need for headers and the code duplication they entail.
 
+To learn more about the programming language, visit [swift.org](https://swift.org/documentation/).
 
-## Documentation
-
-To read the documentation, start by installing the
-[Sphinx](http://sphinx-doc.org) documentation generator tool by running the command:
-
-`easy_install -U Sphinx`
-
-Once complete, you can build the Swift documentation by changing directory into
-[docs](https://github.com/apple/swift/tree/master/docs) and typing `make`. This
-compiles the `.rst` files in the [docs](https://github.com/apple/swift/tree/master/docs)
-directory into HTML in the `docs/_build/html` directory.
+## Contributing to Swift
 
-Many of the docs are out of date, but you can see some historical design
-documents in the `docs` directory.
+Contributions to Swift are welcomed and encouraged! Please see the
+[Contributing to Swift guide](https://swift.org/contributing/).
 
-Another source of documentation is the standard library itself, located in
-`stdlib`. Much of the language is actually implemented in the library
-(including `Int`), and the standard library gives some examples of what can be
-expressed today.
+To be a truly great community, [Swift.org](https://swift.org/) needs to welcome
+developers from all walks of life, with different backgrounds, and with a wide
+range of experience. A diverse and friendly community will have more great
+ideas, more unique perspectives, and produce more great code. We will work
+diligently to make the Swift community welcoming to everyone.
 
+To give clarity of what is expected of our members, Swift has adopted the
+code of conduct defined by the Contributor Covenant. This document is used
+across many open source communities, and we think it articulates our values
+well. For more, see the [Code of Conduct](https://swift.org/community/#code-of-conduct).
 
 ## Getting Started
 
-These instructions give the most direct path to a working Swift
-development environment. Options for doing things differently are
-discussed below.
+These instructions give the most direct path to a working Swift development
+environment. To build from source you will need 2 GB of disk space for the
+source code and over 20 GB of disk space for the build artifacts. The first
+build can take multiple hours, but incremental builds will finish much faster.
 
 
 ### System Requirements
 
 macOS, Ubuntu Linux LTS, and the latest Ubuntu Linux release are the current
 supported host development operating systems.
 
-For macOS, you need [Xcode 9.0 beta 4](https://developer.apple.com/xcode/downloads/).
+<a name="xcode"></a>
```

Comment:
Formatting nitpick: I think this would be a bit clearer with a `#### macOS` header to go balance with `#### Linux` below. Or maybe remove the Linux header and just rely on the paragraph separation of sections.